package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "dev": "./node_modules/.bin/nodemon ./bin/www",
-    "dev:debug": "./node_modules/.bin/nodemon --inspect-brk ./bin/www",
+		"dev": "./node_modules/.bin/nodemon src/app.js",
```

Comment:
thank you for updating this! looks like something funky might be going on with indentation?